dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -203,8 +203,7 @@ public void testLinkageReports() {
     Nodes reports = details.query("//p[@class='jar-linkage-report']");
     // appengine-api-sdk, shown as first item in linkage errors, has these errors
     Truth.assertThat(trimAndCollapseWhiteSpace(reports.get(0).getValue()))
-        .isEqualTo(
-            "106 target classes causing linkage errors referenced from 516 source classes.");
+        .isEqualTo("115 symbols causing linkage errors referenced from 1,648 classes.");
```

Comment:
why did these numbers change?

Reply:
With new value class, it counts linkage errors more uniquely.

A symbol problem caused by CLASS_NOT_FOUND has a ClassSymbol regardless of its origin (class reference, method reference, or field reference). This helps uniquely count symbol problems.